jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -934,11 +934,11 @@ ecma_builtin_json_stringify (ecma_value_t this_arg, /**< 'this' argument */
                                    array_length,
                                    ret_value);
 
-      uint32_t array_length = ecma_number_to_uint32 (array_length_num);
+      uint32_t conv_array_length_num = ecma_number_to_uint32 (array_length_num);
       uint32_t index = 0;
 
       /* 4.b.ii */
-      while ((index < array_length) && ecma_is_value_empty (ret_value))
+      while ((index < conv_array_length_num) && ecma_is_value_empty (ret_value))
```

Comment:
If this is used only once, we could put the ecma_number_to_uint32 (array_length_num) here.